client/my-sites/promote-post-i2/components/campaign-item-details/index.tsx

```diff
@@ -629,63 +629,63 @@ export default function CampaignItemDetails( props: Props ) {
 											</>
 										) }
 									</div>
-								</div>
```

Comment:
Just moved this div after the campaign stats, so they now share the same container